### PR TITLE
Db 697 sql file a

### PR DESCRIPTION
**Pull request by jworcestBAH**
Includes all 'A' rules as sql

### CODE REVIEW

**Comment by jworcestBAH** [comment]
This includes material from PR 122: https://github.com/fedspendingtransparency/data-act-broker-backend/pull/122

File: dataactvalidator/config/sqlrules/a21_appropriations.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+SELECT
+    row_number,
+    availability_type_code
+FROM appropriation
+WHERE submission_id = {}
+AND LOWER(COALESCE(availability_type_code,'')) NOT IN ('X','x','')
```
Comment (bsweger) [inline dataactvalidator/config/sqlrules/a21_appropriations.sql]: Now that we're using `LOWER`, we don't need `X` in the list of acceptable values.

**Comment by bsweger** [comment]
@jworcestBAH This all looks good. One comment about `x` and `X` (see above) and also wanted to confirm that the reason for the file size differences in the unit tests are because we're still using the short names.

File: dataactvalidator/config/sqlrules/a21_appropriations.sql (hunk 1 above)
Comment (jworcestBAH) [inline dataactvalidator/config/sqlrules/a21_appropriations.sql]: Good catch, that got left in from experimenting with different forms, it's now been removed.

**Comment by jworcestBAH** [comment]
Difference in number of errors and file size change is primarily because A21 had not been applied to files B and C in the original definitions.  Prac & proc specifies that it applies to all three files, so I added it as a sql rule for all three, which brings in the new errors.

**Comment by bsweger** [comment]
Looks good--merging!

**Pull request merged**